pyproject.toml

```diff
@@ -61,6 +58,21 @@ nats-py = "^2.7.2"
 netaddr = "1.3.0"
 authlib = "1.3.2"
 
+
+# Dependencies specific to the SDK
+Jinja2 = "^3"
+gitpython = "^3"
+pyyaml = "^6"
+toml = "^0.10"
```

Comment:
I think this is fine for now during the transfer but it's not completely correct as we're using gitpython within Infrahub directly, so we should do a review and see what we actually need within Infrahub itself.

Reply:
Yes this is a good point, I moved Jinja2, gitpython, pyyaml and toml to the main section, as far as I know all 4 of them are used in Infrahub directly